### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by mohamedelshamy55**
Use basic JavaScript syntax.
Use JavaScript to manipulate DOM elements.
Use JavaScript events.
Understand how to parse a Figma design to create a UI.
Use objects to store and access data.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -86,7 +86,7 @@ <h2>Tonic</h2>
                         <li>css</li>
                         <li>javascript</li>
                     </ul>
-                    <button class="project-btn" type="button">See Project</button>
+                    <button class="project-btn btn" type="button" data-btn-id="1">See Project</button>
```
Comment (rotimiazeez) [inline index.html:89]: - [ ] Kindly know that the projects section should be created dynamically with JavaScript for this milestone🙄, as it enables you to remove all hard-coded content and it makes it easier to add more projects later.😊😎

File: index.html [hunk 2]
```diff
@@ -161,7 +161,7 @@ <h2>Uber Navigation</h2>
                         <li>css</li>
                         <li>javascript</li>
                     </ul>
-                    <button class="project-btn" type="button">See Project</button>
+                    <button class="project-btn btn" type="button" data-btn-id="4">See Project</button>
```
Comment (rotimiazeez) [inline index.html:164]: Kindly note that the popup follows the design for both desktop and mobile versions from the chosen template in Figma (e.g., font, colours, images, tests, margins). 👌✔
- Figma desktop design
screenshot (1)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @mohamedelshamy55 👋, 

### Good job so far👏👏👏

### To Highlight
- No Linter errors ✔
- Good UI design 👍
- Good README 😎

There are some issues that you still need to work on to go to the next project but you are almost there!👍💯.
### Great job so far 👍 just a few changes and you are on to the next level😎💯

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mohamedelshamy55) [reply index.html:89]: Please, can you explain more what do you mean?

File: index.html (hunk 2 above)
Comment (mohamedelshamy55) [reply index.html:164]: Please, can you explain more what do you mean?

File: index.html (hunk 1 above)
Comment (mohamedelshamy55) [reply index.html:89]: Please, can you explain more what do you mean?

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @mohamedelshamy55 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**